### DISCUSSION TITLE
Add hardware aceleration support

### DISCUSSION THREAD

**Discussion by JaviLerma** [tteck/Proxmox#3318] [Ideas]
Hi,

I would like to ask if it is possible to create a script to add hardware acceleration support to an unprivileged LXC that has Docker, so that Immich can use it. 

Previously, I installed Plex Server LXC on my Proxmox (AMD iGPU), and the hardware acceleration provided by the script works wonderfully. 
Thanks!

**Comment by tteck** [comment]
This [discussion](https://github.com/tteck/Proxmox/discussions/2850) may help
